specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.0/TextAnalytics.json

```diff
@@ -572,5 +576,16 @@
         }
       }
     }
+  },
+  "parameters": {
+    "Endpoint": {
+      "name": "Endpoint",
+      "description": "Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus.api.cognitive.microsoft.com).",
```

Comment:
Hi @assafi, we will update Azure Portal accordingly when we publish SDK.
As in description, I emphasized `protocol and hostname` and attached an example. Any suggestion how to make it more clear?

What which document you refer to?
- If you refer to API Reference (https://docs.microsoft.com/en-us/rest/api/cognitiveservices/textanalytics/detect%20language/detect%20language), they are auto generated by this swagger, once this get checked in, the document will be automatically updated.
- If you refer to https://github.com/MicrosoftDocs/azure-docs, I think it should be updated along with SDK PR.

Reply:
What do you mean by updating the Azure portal? 

I'm referring to the [TA documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/quickstarts/csharp#call-the-text-analytics-api-using-the-sdk) from your previous PR, which says:

>Replace the location in client.BaseUri to the endpoint you signed up for. You can find the endpoint on Azure Portal resource. The endpoint typically looks like "https://[region].api.cognitive.microsoft.com/text/analytics/v2.0".

If you will be using a similar language to describe this you should note that the endpoint from the Azure portal contains a suffix which you exclude in this PR.  

Ideally, we should update the SDK and its documentation at the same time, to avoid recent issues like lagging documentation. Which is why I asked the description of this PR to contain a link to the documentation PR.

Reply:
I will change Azure Portal resource page to remove the suffix to keep consistent. The document will also say:

> You can find the endpoint on Azure Portal resource. The endpoint typically looks like "https://[region].api.cognitive.microsoft.com/"

Does it make sense?

About document PR, I agree SDK (actually SDK package) and its documentation should be updated at the same time. But this is swagger PR?
After merge swagger PR, I will send C# SDK PR. I prefer send the documentation PR along with C# SDK PR, because at that time I can test the code in the documents. Also the document won't be accidentally published before NuGet (sometimes you say "please hold", but...)

We don't need to worry about the lagging documentation. It is hard to control when documents get updated, but we can control when to publish NuGet.

Reply:
Regarding the portal update - I don't think we need to change it. Not all users use the SDK, some simply write their own code in whatever language they want and they will need the full URL. It's important not to break existing or future customers just because we changed how a parameter looks in the SDK (because of some other swagger limitation). The solution for this should be better documentation and/or smarter SDK. 

Regarding the documentation PR - sure, please attach it to the SDK PR.

Reply:
Moved to offline...

Reply:
We've agreed to go with `baseEndpoint` instead of `Endpoint` at a later update.